crates/uv/src/commands/project/init.rs

```diff
@@ -829,6 +885,212 @@ fn pyproject_project_scripts(package: &PackageName, executable_name: &str, targe
     "#}
 }
 
+/// Generate additional files as needed for specific build backends.
+fn pyproject_build_backend_prerequisites(
+    package: &PackageName,
+    path: &Path,
+    build_backend: ProjectBuildBackend,
+) -> Result<()> {
+    let module_name = package.as_dist_info_name();
+    match build_backend {
+        ProjectBuildBackend::Maturin => {
+            // Generate Cargo.toml
+            let build_file = path.join("Cargo.toml");
+            if !build_file.try_exists()? {
+                fs_err::write(
+                    build_file,
+                    indoc::formatdoc! {r#"
+                    [package]
+                    name = "{module_name}"
+                    version = "0.1.0"
+                    edition = "2021"
+
+                    [lib]
+                    name = "_core"
+                    # "cdylib" is necessary to produce a shared library for Python to import from.
+                    crate-type = ["cdylib"]
+
+                    [dependencies]
+                    # "extension-module" tells pyo3 we want to build an extension module (skips linking against libpython.so)
+                    # "abi3-py38" tells pyo3 (and maturin) to build using the stable ABI with minimum Python version 3.8
+                    pyo3 = {{ version = "0.22.3", features = ["extension-module", "abi3-py38"] }}
```

Comment:
we're just in time to switch to 3.9 as starter default minimum version (https://devguide.python.org/versions/)

Reply:
♥️

Reply:
✔️ a090e8f286e41bb2be8d81999d19eb3571fc4bda